core/templates/dev/head/components/correct_editor_directive.html

```diff
@@ -0,0 +1,38 @@
+<div ng-if="!correctEditorIsOpen"
+title="<[isEditable() ? 'Edit correctness setting' : '']>"
+style="height: 100%;">
+  <div class="oppia-readonly-rule-tile"
+    ng-class="{'oppia-editable-section': isEditable()}">
+    <div class="oppia-rule-details-header">
+       <div class="oppia-click-to-start-editing"
+        ng-if="isEditable()" ng-click="openCorrectEditor()">
+         <i ng-if="isEditable()" class="material-icons oppia-editor-edit-icon pull-right"
+          title="Edit Correctness setting">&#xE254;
+         </i>
+       </div>
+
+       <strong>The answers falling under this group are
+        <span ng-if="labelledAsCorrect" style="color: rgb(20, 180, 15)">
```

Comment:
Indent by 2, not 1. Please check your indentation throughout this file; going from a tag to a child tag should result in an additional indentation of 2 spaces.

Reply:
Done!